docs/conceptual/core/consensus.md

```diff
@@ -2,14 +2,104 @@
 title: Consensus
 ---
 
```

Comment:
Please add a small 1-2 sentences telling the user what they will find on this page.